### PR TITLE
Installation with Docker and Openhab2

### PR DESCRIPTION
**Pull request by matte150670**
Installation with Docker on a Ubuntu System with Back End Openhab2.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "restructuredtext.confPath" : "${workspaceRoot}/doc/manual/de",
+    "restructuredtext.confPath" : "${workspaceFolder}/doc/manual/de",
```
Comment (peuter) [inline .vscode/settings.json:2]: Is this a required change to fuix the setup or something the visual studio code has changed by itself?

File: source/editor/lib/DocumentationMapping.json [hunk 2]
```diff
@@ -1,41 +1,48 @@
...
+  "xml-format": "/develop/manual/config/xml-format.html#xml-format", 
+  "xml-format-files": "/develop/manual/config/xml-format.html#xml-format-files", 
+  "xml-format-header": "/develop/manual/config/xml-format.html#xml-format-header", 
+  "xml-format-icons": "/develop/manual/config/xml-format.html#xml-format-icons", 
+  "xml-format-mappings": "/develop/manual/config/xml-format.html#xml-format-mappings", 
+  "xml-format-pages": "/develop/manual/config/xml-format.html#xml-format-pages", 
+  "xml-format-plugins": "/develop/manual/config/xml-format.html#xml-format-plugins", 
+  "xml-format-statusbar": "/develop/manual/config/xml-format.html#xml-format-statusbar", 
+  "xml-format-stylings": "/develop/manual/config/xml-format.html#xml-format-stylings", 
+  "xml-format-templates": "/develop/manual/config/xml-format.html#xml-format-templates"
```
Comment (peuter) [inline source/editor/lib/DocumentationMapping.json:47]: Please remove the changes for this file from the PR. The file is generated when you build the docs, but the change must not be commited.

File: .vscode/settings.json (hunk 1 above)
Comment (matte150670) [reply .vscode/settings.json:2]: changed by vs code

File: source/editor/lib/DocumentationMapping.json (hunk 2 above)
Comment (matte150670) [reply source/editor/lib/DocumentationMapping.json:47]: I have no experience with that. Can you support this? I have commited all changes in VS Code.

**Pull request merged**